invenio_communities/members/services/config.py

```diff
@@ -91,6 +113,26 @@ class MemberSearchOptions(PublicSearchOptions):
         'visibility': facets.visibility,
     }
 
+    query_parser_cls = QueryParser.factory(
+        fields=[
+            "user.username^2",
```

Comment:
basically search will **only** happen over username, email, full_name and affiliations.